script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt

```diff
@@ -449,7 +449,7 @@ Vampires that have sired a bunch of Dead usually spend most of their time sleepi
 [sha:e6d75c9ab9aa0b53f30affdd4220954b64b9367d]{
 -- Page 546, Offset 17936.
 -- 長く生きた吸血鬼は燃費が悪いから、だとか。なので頻繁には動かず、自分が何も<消|つ><費|か>わなくとも<血液|エネルギー>を集める手足を必要とする。
-As I understand it, vampires that have been alive for a long time aren't very fuel efficient.
+As I understand it, vampires that have been alive for a long time aren't very fuel-efficient.
```

Comment:
It shouldn't be hyphenated, we discussed this instance before.

Reply:
we did? i thought if it's used as an adjective it should be hyphenated

Reply:
that was my opinion too

Reply:
hmm so what was the reason for not hyphenating it then?

Reply:
From some example I've seen, it should only be hyphenated when followed by a noun.

> Fuel-efficient cars.

> The car isn't very fuel efficient.

Reply:
we talked about this during group PR and settled on fuel efficient non-hyphenated... I think it was an American english vs. english english question but it's been quite some time.